### PR TITLE
refactor(path): format  test names

### PR DESCRIPTION
**Pull request by timreichen**
Ref: https://github.com/denoland/deno_std/issues/3754

I added `posix` and `win32` as prefixes to the function name (for example ` posix.toFileUrl()`).

Question: lots of tests do not have a description. Most of these would repeat the information in the function name. Do all tests need a description?

### CODE REVIEW

File: path/glob_test.ts [hunk 1]
```diff
@@ -370,17 +370,17 @@ Deno.test({
...
     assertEquals(globToRegExp("[!?*+@!|]", { os: "linux" }), /^[^?*+@!|]\/*$/);
   },
 });
 
 Deno.test({
-  name: "[path] globToRegExp() Special RegExp characters in range",
+  name: "globToRegExp() matches special RegExp characters in range",
   fn() {
-    // Excluding characters checked in the previous test.
+    // Excluding characters checksed in the previous test.
```
Comment (kt3k) [inline path/glob_test.ts:383]:
```suggestion
    // Excluding characters checked in the previous test.
```

**Review by kt3k — APPROVED** [review]
LGTM. Thanks!

**Comment by kt3k** [comment]
> Question: lots of tests do not have a description. Most of these would repeat the information in the function name. Do all tests need a description?

Can you provide an example?

**Comment by timreichen** [comment]
> > Question: lots of tests do not have a description. Most of these would repeat the information in the function name. Do all tests need a description?
> 
> Can you provide an example?

For example in `basename_test.ts`: There are only two tests (`posix.basename()` and `posix.basename()`) I guess the description would be something like `returns the basename of a posix path` and `returns the basename of a win32 path`  respectively.

**Comment by kt3k** [comment]
I think it's fine to omit the description in such cases

**Pull request merged**